templates/typescript/src/components/typography/style.scss

```diff
@@ -1,15 +1,15 @@
-#H1 {
+.H1 {
```

Comment:
In terms of specificity. If we use a class instead of an ID, we'd override all elements that are using this class. Toch?